python/ribasim/tests/test_edge.py

```diff
@@ -16,6 +17,10 @@ def test():
     edge = Edge(static=df)
     assert isinstance(edge, Edge)
 
+    # Plotting
```

Comment:
Let's give this test a better name so that it is clear that we test whether validation works.

Then make a separate test which tests whether plotting works.